doc/v2/documentation_data_format.rst

```diff
@@ -309,8 +419,77 @@ Detailed field description
 
 
-Observable table
-----------------
+.. _experiments_table:
+
+Experiments table
+-----------------
+
+The optional experiments table describes a sequence of different experimental
+conditions (here: discrete changes) that are applied to the model.
+
+This is specified as a tab-separated value file in the following way:
+
++---------------------+-------------------+-----------------+
+| experimentId        | time              | conditionId     |
++=====================+===================+=================+
+| PETAB_ID            | NUMERIC or '-inf' | CONDITION_ID    |
++---------------------+-------------------+-----------------+
+| timecourse_1        |                 0 | condition_1     |
++---------------------+-------------------+-----------------+
+| timecourse_1        |                10 | condition_2     |
++---------------------+-------------------+-----------------+
+| timecourse_1        |               250 | condition_3     |
++---------------------+-------------------+-----------------+
+| patient_3           |              -inf | condition_1     |
++---------------------+-------------------+-----------------+
+| patient_3           |                 0 | condition_2     |
++---------------------+-------------------+-----------------+
+| intervention_effect |               -20 | no_lockdown     |
++---------------------+-------------------+-----------------+
+| intervention_effect |                20 | mild_lockdown   |
++---------------------+-------------------+-----------------+
+| intervention_effect |                40 | severe_lockdown |
++---------------------+-------------------+-----------------+
+
+Detailed field description
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The time courses table with three mandatory columns ``experimentId``,
+``time``, and ``conditionId``:
+
+- ``experimentId`` [STRING, REQUIRED]
+
+  Identifier of the experiment. The usual PEtab identifier requirements apply.
+  This is referenced by the ``experimentId`` column in the measurement table.
+
+- ``time``: [NUMERIC or  ``-inf``, REQUIRED]
+
+  The time when the condition will become active, in the time unit specified
+  in the model. ``-inf`` indicates pre-equilibration (e.g., for drug
+  treatments, the model would be pre-equilibrated with the no-drug condition).
+
+- ``CONDITION_ID``: Reference to a condition ID in the condition table that
+  is to be applied at the given `time`.
+
+  Note: The time interval in which a condition is applied includes the
+  respective starting timepoint, but excludes the starting timepoint of
+  the following condition. This means that for an experiment
+  ``[time_A:condition_A; time_B:condition_B]``, ``condition_A`` is active
+  during the interval ``[time_A, time_B)``. This implies that any event
+  assignment that triggers at ``time_B`` will occur *after* ``condition_B`` was
+  applied and for any measurements at ``time_B``, the observables will be
+  evaluated *after* ``condition_B`` was applied.
```

Comment:
This is the old (not long format) version of the experiments table, so it might be confusing to users

```suggestion
  Note: The time interval in which a condition is applied includes the
  respective starting timepoint, but excludes the starting timepoint of
  the following condition. This means that for an experiment where
  ``condition_A`` is applied at ``time_A`` and later ``condition_B`` is
  applied at ``time_B``, then ``condition_A`` is active
  during the interval ``[time_A, time_B)``. This implies that any event
  assignment that triggers at ``time_B`` will occur *after* ``condition_B`` was
  applied and for any measurements at ``time_B``, the observables will be
  evaluated *after* ``condition_B`` was applied.
```